### PR TITLE
Atom 3843 validate port descriptions

### PR DESCRIPTION
**Pull request by dougperez**

### CODE REVIEW

**Comment by captainbeardo** [comment]
:+1:

File: spec/conduit/sprint/actions/validate_port_spec.rb [hunk 1]
```diff
@@ -39,7 +39,11 @@
         :mdn                                    => "5555555555",
         :message                                => "PTN is eligible for Porting In",
         :number_portability_direction_indicator => "C",
+        :number_portability_direction_indicator_description => "Wireline to Wireless",
         :old_service_provider                   => "Neutral Tandem",
+        :port_id                                => nil,
+        :port_in_status                         => nil,
+        :port_in_status_text                    => nil,
```
Comment (pedromartinez) [inline spec/conduit/sprint/actions/validate_port_spec.rb]: I'd be nice to add examples here

File: lib/conduit/sprint/parsers/validate_port.rb [hunk 2]
```diff
@@ -12,6 +17,18 @@ class ValidatePort::Parser < Parser::Base
       content_for '//ppvStatusText'
     end
 
+    attribute :port_in_status do
```
Comment (pedromartinez) [inline lib/conduit/sprint/parsers/validate_port.rb]: Sweet, thanks for adding these extra attributes!

File: spec/conduit/sprint/actions/validate_port_spec.rb (hunk 1 above)
Comment (dougperez) [inline spec/conduit/sprint/actions/validate_port_spec.rb]: Added.

**Comment by pedromartinez** [comment]
:+1:

**Pull request merged**